Description: Correct pod NAME issue in LedgerSMB/File.pm
 Has the module name but not the dash plus short description in the NAME entry.
Bug: http://sourceforge.net/tracker/?func=detail&aid=3472581&group_id=175965&atid=1005191
Author: Robert James Clay <jame@rocasa.us>

--- a/LedgerSMB/File.pm
+++ b/LedgerSMB/File.pm
@@ -2,7 +2,7 @@
 
 =head1 NAME
 
-LedgerSMB::File
+LedgerSMB::File - Provides routines for managing file attachments.
 
 =head1 SYNPSIS
 
